packages/preview/guided-resume-starter-cgc/2.0.0/template/templates/resume.template.typ

```diff
@@ -0,0 +1,148 @@
+#let resume(
```

Comment:
Unless you expect people to edit the contents of these functions, I think this file should be outside of the `template/` folder (either directly in the root of the package, or in a dedicated sub-folder). As it is done now, this template file would be duplicated in every new project initialized from this template.

Reply:
This project is a bit more "to each their own" than something like IEEE-compliant formatting, so the hope was to include the template file directly to more easily allow users to tweak it to their comfort. However, I'd imagine that Typst is looking to encourage adoption for the packaging system.

As a comfortable compromise, in [`49d62cd`](https://github.com/typst/packages/pull/580/commits/f9d62cdadd5ab001bb625970125161d153255840) I've kept the local copy of the template in the repository, switched the `#import` to the package system copy, and provided a brief instruction for users to use the local copy to tweak the editing if they'd like.

I've also added additional comments to the template file itself to make it a bit friendlier to power users who do want to tinker around with the template directly

Hopefully this is a comfortable in-between!

Reply:
It is, thank you :)